luaui/Widgets/gui_gridmenu.lua

```diff
@@ -104,28 +109,27 @@ local myTeamID = Spring.GetMyTeamID()
 local startDefID = Spring.GetTeamRulesParam(myTeamID, "startUnit")
```

Comment:
nitpick
Is there any reason to cache this value this early when it's updated in both init and pregame update logic?

Reply:
Not really, good catch

Reply:
code outside of any widget:XXXX() function does get run like... two times on startup

once to load it up so that widget:GetInfo() can be called and potentially another time to actually Initialize the widget

I'd have to check the widget loading code to make sure, been a while since I've looked at it